lib/govspeak.rb

```diff
@@ -269,11 +269,18 @@ def render_image(image)
       lines = []
       lines << %{<figure#{id_attr} class="image embedded">}
       lines << %{<div class="img"><img src="#{encode(image.url)}" alt="#{encode(image.alt_text)}"></div>}
-      lines << %{<figcaption>#{encode(image.caption)}</figcaption>} if image.caption
+      lines << '<figcaption>' if figcaption?(image)
```

Comment:
minor but since the subsequent lines inter join with the condition it might be clearer to just do all the figcaption stuff inside the condition. Which might mean you can drop the `figcaption?` method as I feel that's quite specific to be part of this `Govspeak::Document` class and more the domain of the presenter

Reply:
Fixed-up!

Reply:
I've made this change. It's in a separate commit at the moment, but I will fix it up.

Reply:
Updated and fixed up.